src/bin/bors.rs

```diff
@@ -72,14 +72,14 @@ fn try_main(opts: Opts) -> anyhow::Result<()> {
         .block_on(initialize_db(&opts.db))
         .context("Cannot initialize database")?;
 
-    let state = runtime
-        .block_on(GithubAppState::load(
-            opts.app_id.into(),
-            opts.private_key.into_bytes().into(),
-        ))
-        .context("Cannot load GitHub repository state")?;
-    let ctx = BorsContext::new(CommandParser::new(opts.cmd_prefix), Arc::new(db));
-    let (repository_tx, global_tx, bors_process) = create_bors_process(state, ctx);
+    let client = create_github_client(opts.app_id.into(), opts.private_key.into_bytes().into())?;
```

Comment:
This crashes at runtime, it needs to be executed within a tokio Runtime. Let's move it into an `async move` block in the `block_on` call below.